setup.py

```diff
@@ -72,18 +69,7 @@
     ":python_version<'3.5'": ['contextlib2', 'typing'],
 }
 
-if int(setuptools.__version__.split('.')[0]) < 18:
```

Comment:
@kotfu 
setuptools 18 dates back to early/mid 2015, so I don't think we really need to worry about people using a version prior to this.  If you think I'm off-base, please feel free to revert this.

I probably should have put this on master, but you made some changes to it as well so I didn't want to create any conflicts.